src/Entity/BaseEvent.php

```diff
@@ -53,6 +54,9 @@
      * @ORM\Column(length=100)
      *
      * @Algolia\Attribute
+     *
+     * @JMS\Groups({"public"})
+     * @JMS\SerializedName("name")
```

Comment:
Why do you need to use SerializedName ? It isn't by default the property name  ?

Reply:
indeed. just a habit to explicitely declare this.
I will remove all default/useless declaration